package.json

```diff
@@ -55,6 +57,7 @@
     "eslint-plugin-simple-import-sort": "8.0.0",
     "eventemitter3": "4.0.7",
     "execa": "5.1.1",
+    "fs-extra": "10.1.0",
```

Comment:
Why do we need `fs-extra` here? The only usage I see is `writeFileSync`, which is also available in builtin module.

Reply:
Just a habit of always using fs-extra.